runtime/westend/src/lib.rs

```diff
@@ -100,6 +100,7 @@ pub use pallet_staking::StakerStatus;
 pub use sp_runtime::BuildStorage;
 pub use pallet_timestamp::Call as TimestampCall;
 pub use pallet_balances::Call as BalancesCall;
+pub use pallet_election_provider_multi_phase::Call as EPMCall;
```

Comment:
Is this used? Same goes for polkadot + kusama runtimes

Reply:
yes, in `create_uxt_$runtime` function.